Sources/BaggageLogging/LoggingBaggageContextCarrier.swift

```diff
@@ -27,7 +27,22 @@ public protocol LoggingBaggageContextCarrier: BaggageContextCarrier {
     /// SHOULD implement this logger by wrapping the "raw" logger associated with this context with the `logger.with(BaggageContext:)` function,
     /// which efficiently handles the bridging of baggage to logging metadata values.
     ///
+    /// ### Example implementation
+    ///
     /// Writes to the `logger` metadata SHOULD NOT be reflected in the `baggage`,
     /// however writes to the underlying `baggage` SHOULD be reflected in the `logger`.
+    ///
+    ///     struct MyFrameworkContext: LoggingBaggageContextCarrier {
+    ///       var baggage = BaggageContext()
+    ///       private let _logger: Logger
+    ///
+    ///       var logger: Logger {
+    ///         get {
+    ///           return self._logger.with(context: self.baggage)
+    ///         }
+    ///         set {
+    ///           self._logger = newValue
+    ///         }
```

Comment:
maybe remove the `set` (see my other PR on this)

OR

make it a separate snippet where we say: 

```
/// Optionally, the underlying logger may be `set` as well, to allow users to e.g. the log level of the used logger.
```

Reply:
I think that composes right... If I'm a framework I should not necessarily change the log level of a logger that was passed to me -- I.e. I was passed a generic Carrier.

But if I'm user land and I have the specific framework context, everything goes and I may want to change the logger hmm...

I wonder if we're overdoing it or being helpful though

Reply:
(Need to think more about that PR https://github.com/slashmo/gsoc-swift-baggage-context/pull/31 what REALLY composes and what's annoying)